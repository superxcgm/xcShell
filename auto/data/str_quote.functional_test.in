echo "hello world"
echo 'hello world'
echo "hello world" 'hi ni'
echo hello world
echo "j's'f"