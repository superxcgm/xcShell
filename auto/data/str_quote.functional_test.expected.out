xcShell_functional_test > echo "hello world"
hello world
xcShell_functional_test > echo 'hello world'
hello world
xcShell_functional_test > echo "hello world" 'hi ni'
hello world hi ni
xcShell_functional_test > echo hello world
hello world
xcShell_functional_test > echo "j's'f"
j's'f
xcShell_functional_test > echo $haha

xcShell_functional_test > echo $EXIST_PATH_ENV
hello
xcShell_functional_test > echo 123$EXIST_PATH_ENV
123hello
xcShell_functional_test > echo $123EXIST_PATH_ENV
EXIST_PATH_ENV
xcShell_functional_test > echo $EXIST_PATH_ENV.123
hello.123
xcShell_functional_test > echo $EXIST_PATH_ENV123

xcShell_functional_test > echo $123EXIST_PATH_ENV.123
EXIST_PATH_ENV.123
xcShell_functional_test > echo 123$EXIST_PATH_ENV.123
123hello.123
xcShell_functional_test > echo "$EXIST_PATH_ENV"
hello
xcShell_functional_test > echo $EXIST_PATH_ENV $EXIST_PATH_ENV_TEST
hello world
xcShell_functional_test > echo "$EXIST_PATH_ENV $EXIST_PATH_ENV_TEST"
hello world
xcShell_functional_test > echo "$EXIST_PATH_ENV $haha"
hello 
xcShell_functional_test > echo $EXIST_PATH_ENV $haha
hello 
xcShell_functional_test > echo "HOME""PATH"
HOMEPATH
xcShell_functional_test > echo "$$EXIST_PATH_ENV""$EXIST_PATH_ENV_TEST"
helloworld
xcShell_functional_test > 